etc/cicd/inputs.conf

```
@@ -55,3 +55,15 @@ account = mock
 interval = 900
 disabled = 1
 index = genesyscloud
+
+[actions_metrics://actions_metrics]
+account = mock
+index = genesyscloud
+interval = 900
+disabled = 1
+
+[audit_query://audit_query]
+account = mock
+index = genesyscloud
+interval = 900
+disabled = 1
```

Aren't the 2 new parameters missing here? 🤔 
```suggestion
disabled = 1
poll_interval_seconds = 2 
max_poll_attempts = 10
```